training_config/biattentive_classification_network_elmo.json

```diff
@@ -0,0 +1,98 @@
+{
+  "dataset_reader":{
+    "type": "sst_tokens",
+    "use_subtrees": true,
+    "granularity": "5-class",
+    "token_indexers": {
+      "tokens": {
+        "type": "single_id"
+      },
+      "elmo": {
+        "type": "elmo_characters"
+      }
+    }
+  },
+  "validation_dataset_reader":{
+    "type": "sst_tokens",
+    "use_subtrees": false,
+    "granularity": "5-class",
+    "token_indexers": {
+      "tokens": {
+        "type": "single_id"
+      },
+      "elmo": {
```

Comment:
Given this is HOCON, can you add a descriptive comment about why this doesn't break everything, like you have done in the testing configs?